etl/nixpkgs-graph.nix

```diff
@@ -1,17 +1,38 @@
 # usage:
-# NIXPKGS_FLAKE_REF="github:nixos/nixpkgs/master" nix eval --json --file "./nixpkgs-graph.nix"
+# NIXPKGS_ALLOW_BROKEN=1 NIXPKGS_ALLOW_INSECURE=1 TARGET_FLAKE_REF="github:nixos/nixpkgs/master" TARGET_FLAKE_SYSTEM="x86_64-linux" nix eval --json --file "./nixpkgs-graph.nix"
 
 let
-  nixpkgsFlakeRef = builtins.getEnv "NIXPKGS_FLAKE_REF";
-  pkgs = import (builtins.getFlake nixpkgsFlakeRef) {
-    config = {
-      # package 'python-2.7.18.6' is marked as insecure
-      # we need to allow this package otherwise it is refusing to evaluate
-      permittedInsecurePackages = [
-        "python-2.7.18.6"
-      ];
-    };
-  };
+  pkgs = builtins.getFlake "nixpkgs";
+  targetFlakeRef = builtins.getEnv "TARGET_FLAKE_REF";
+  # We specify the system of the target flake here to avoid ambiguity
+  # An output will produce a different result on every system
```

Comment:
What do you mean by "an output"?

Reply:
An output of the flake, say `(builtins.getFlake github:nixos/nixpkgs/master).outputs.legacyPackages` on different systems.